raft/tracker/inflights.go

```diff
@@ -14,6 +14,12 @@
 
 package tracker
 
+// inflight describes an in-flight message.
```

Comment:
More precisely, an in-flight `MsgApp`. I think that bit of clarification can be helpful to readers less in the weeds than us.